tooling/docker/dev/set_up_ubuntu.sh

```diff
@@ -12,6 +12,11 @@ apt-get -q update \
         llvm-3.5 libclang-3.5-dev clang-3.5 \
         curl
 
+# Install newer node.
+apt-get remove -y nodejs
+curl -sL https://deb.nodesource.com/setup_6.x | bash -
```

Comment:
Not a blocker for this PR, but we should see if we can get the various curl-to-root-shell commands out of here. Since we've moved from a proper VM to Docker, our containment is worse.